.teamcity/settings.kts

```diff
@@ -1224,12 +1234,10 @@ object WPComPlugins_EditorToolKit : BuildType({
 
 	triggers {
 		vcs {
-			triggerRules = "+:apps/editing-toolkit/**"
```

Comment:
@scinos, I'm getting a warning here in TeamCity: `Unsupported change of build triggers in the build configuration 'Calypso / WPCom Plugins / Editor ToolKit' has been detected in the settings taken from VCS, the current settings from TeamCity server will be used instead`.

I'm not sure why, as this "triggers" field should now match some of the other builds we have 🤔

Reply:
I'm not sure, but I've seen several times that TeamCity doesn't like when you change the build config (specially the `features` section) in a branch.

I suggest ignore that warning, the change looks good to me.